boards/esp32s3-usb-otg/include/board.h

```diff
@@ -0,0 +1,199 @@
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include <stdint.h>
+
+/**
+ * @name    Button pin definitions
+ * @{
+ */
+
+/**
+ * @brief   BOOT/OK button GPIO pin definition
+ *
+ * The BOOT button is connected to GPIO0 and is used as OK button during
+ * normal operation. The button is LOW active.
+ */
+#define BTN0_PIN        GPIO0
+
+/**
+ * @brief   BOOT/OK button GPIO mode definition
+ */
+#define BTN0_MODE       GPIO_IN_PU
+
+/**
+ * @brief   Default interrupt flank definition for the BOOT/OK button GPIO
+ */
+#ifndef BTN0_INT_FLANK
+#define BTN0_INT_FLANK  GPIO_FALLING
+#endif
+
+/**
+ * @brief   Definition for compatibility with previous versions
+ */
+#define BUTTON0_PIN     BTN0_PIN
+
+/**
+ * @brief   MENU button GPIO pin definition
+ *
+ * The MENU button is connected to GPIO14. It is LOW active.
+ */
+#define BTN1_PIN        GPIO14
+
+/**
+ * @brief   MENU button GPIO mode definition
+ */
+#define BTN1_MODE       GPIO_IN_PU
+
+/**
+ * @brief   Default interrupt flank definition for the MENU button GPIO
+ */
+#ifndef BTN1_INT_FLANK
+#define BTN1_INT_FLANK  GPIO_FALLING
+#endif
+
+/**
+ * @brief   Definition for compatibility with previous versions
+ */
+#define BUTTON1_PIN     BTN1_PIN
+
+/**
+ * @brief   DW- button GPIO pin definition
+ *
+ * The DW- button is connected to GPIO11. It is LOW active.
+ */
+#define BTN2_PIN        GPIO11
+
+/**
+ * @brief   DW- button GPIO mode definition
+ */
+#define BTN2_MODE       GPIO_IN_PU
+
+/**
+ * @brief   Default interrupt flank definition for the DW- button GPIO
+ */
+#ifndef BTN2_INT_FLANK
+#define BTN2_INT_FLANK  GPIO_FALLING
+#endif
+
+/**
+ * @brief   Definition for compatibility with previous versions
+ */
+#define BUTTON2_PIN     BTN2_PIN
+
+/**
+ * @brief   UP+ button GPIO pin definition
+ *
+ * The MENU button is connected to GPIO10. It is LOW active.
+ */
+#define BTN3_PIN        GPIO10
+
+/**
+ * @brief   UP+ button GPIO mode definition
+ */
+#define BTN3_MODE       GPIO_IN_PU
+
+/**
+ * @brief   Default interrupt flank definition for the UP+ button GPIO
+ */
+#ifndef BTN3_INT_FLANK
+#define BTN3_INT_FLANK  GPIO_FALLING
+#endif
+
+/**
+ * @brief   Definition for compatibility with previous versions
+ */
+#define BUTTON3_PIN     BTN3_PIN
+
+/** @} */
+
+/**
+ * @name    LED (on-board) configuration
+ *
+ * ESP32-S3-USB-OTG boards have two LEDs on board.
+ * @{
+ */
+#define LED0_PIN        GPIO15  /**< Green LED */
+#define LED0_ACTIVE     (1)     /**< Green LED is high active */
+
+#define LED1_PIN        GPIO16  /**< Yellow LED */
+#define LED1_ACTIVE     (1)     /**< Yellow LED is high active */
+/** @} */
+
+/**
+ * @name    LCD configuration
+ *
+ * This configuration cannot be changed.
+ * @{
+ */
+#if MODULE_ST7735
+#define LCD_DC              GPIO4   /**< LCD DC signal */
+#define LCD_CS              GPIO5   /**< LCD EN signal */
+#define LCD_RST             GPIO8   /**< LCD RST signal */
+#define LCD_BACKLIGHT       GPIO9   /**< LCD BL signal */
```

Comment:
In #19811 you do a `gpio_init()` for this pin.

Reply:
Good point, since the backlight was working in default state after the reset I didn't notice that the initialization of the backlight is missing.